### PR TITLE
The mxnet shiny app blog

### PR DESCRIPTION
**Pull request by thirdwing**
It is just a draft, so please don't merge it now.

### CODE REVIEW

File: _posts/2015-12-07-image-classification-shiny-app-mxnet-r.markdown [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+```r
+im <- load.image(src)
+normed <- preproc.image(im, mean.img)
+prob <- predict(model, X = normed)
+max.idx <- order(prob[,1], decreasing = TRUE)[1:5]
+result <- synsets[max.idx]
+```
+
+
+If you met any problem, please just open [an issure](https://github.com/dmlc/mxnet/issues). And all PR will be truly appreciated!
```
Comment (tqchen) [inline _posts/2015-12-07-image-classification-shiny-app-mxnet-r.markdown]: add a few more screenshots in the end

File: _posts/2015-12-07-image-classification-shiny-app-mxnet-r.markdown [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+title: "An Image Classification shiny App using MXNetR"
+date:   2015-12-07 13:17:00 -0800
+author: Qiang Kou
+categories: rstats
+comments: true
+---
+
+Early this week, Google announced its [Cloud Vision API](http://googlecloudplatform.blogspot.com/2015/12/Google-Cloud-Vision-API-changes-the-way-applications-understand-images.html), which can detect the content of an image.
+
+It is still in the preview stage and very few people can use it.
```
Comment (tqchen) [inline _posts/2015-12-07-image-classification-shiny-app-mxnet-r.markdown]: Simply remove this line

File: _posts/2015-12-07-image-classification-shiny-app-mxnet-r.markdown [hunk 3]
```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "An Image Classification shiny App using MXNetR"
```
Comment (tqchen) [inline _posts/2015-12-07-image-classification-shiny-app-mxnet-r.markdown]: Build Online Image Classification Service with Shiny and MXNetR

**Comment by phunterlau** [comment]
I can provide some screenshots from `shiny.dmlc.ml`: 
screen shot 2015-11-22 at 12 25 06 am
screen shot 2015-11-22 at 12 19 42 am
screen shot 2015-11-20 at 8 48 09 pm

**Comment by thirdwing** [comment]
@tqchen updated.

**Comment by tqchen** [comment]
LGTM, please check and merge

**Pull request merged**